src/libstd/primitive_docs.rs

```diff
@@ -722,3 +722,35 @@ mod prim_isize { }
 ///
 #[stable(feature = "rust1", since = "1.0.0")]
 mod prim_usize { }
+
+#[doc(primitive = "fn")]
+//
+/// Function pointers, rendered with a signature like `fn(usize) -> bool`.
+///
+/// *See also the traits [`Fn`], [`FnMut`], and [`FnOnce`].*
+///
+/// [`Fn`]: ops/trait.Fn.html
+/// [`FnMut`]: ops/trait.FnMut.html
+/// [`FnOnce`]: ops/trait.FnOnce.html
+///
+/// Plain function pointers are obtained by casting either plain functions, or closures that don't
```

Comment:
It would be useful to mention here somewhere that function pointers are always assumed to be non-null.
This is a common mistake in FFI to use `fn()` in nullable callbacks when `Option<fn()>` need to be used instead.